freshmaker/handlers/koji/rebuild_images_on_rpm_advisory_change.py

```diff
@@ -411,19 +411,19 @@ def _find_images_to_rebuild(self, errata_id):
         if isinstance(self.event, ManualRebuildWithAdvisoryEvent):
             leaf_container_images = self.event.container_images
 
-        # Get srpm nvrs which are affected by the CVEs in this advisory
-        srpm_nvrs = self.event.advisory.affected_srpm_nvrs
+        # Get binary rpm nvrs which are affected by the CVEs in this advisory
+        affected_nvrs = self.event.advisory.affected_rpm_nvrs
 
-        # If there is no CVE affected srpms, this can be non-RHSA advisory,
+        # If there is no CVE affected binary rpms, this can be non-RHSA advisory,
         # just rebuild images that have the builds in this advisory installed
-        if not srpm_nvrs:
-            srpm_nvrs = list(errata.get_builds(errata_id))
+        if not affected_nvrs:
+            affected_nvrs = list(errata.get_builds(errata_id))
```

Comment:
`get_builds` returns the srpm nvrs, since we're changing to filtering by binary rpms, this is also required to be changed or add another api to get the binary rpms.

Reply:
Yes, that was one of questions I wanted to ask you. That method is used in some other places, like **odcsclient.py::prepare_yum_repo** and in **handlers/koji/rebuild_images_on_parent_image_build.py** so I think I should leave the existing implementation with SRPMs for mentioned usecases. And just add new functionality(with binary rpms) to use only in **rebuild_images_on_parent_image_build.py**.
Or should I just change that method to use binary rpms instead or SRPMs? What do you think is better here?

Reply:
And one more thing, there is a comment to the usage of **get_builds** method "# If there is no CVE affected binary rpms, this can be non-RHSA advisory, just rebuild images that have the builds in this advisory installed". So actually as I understand we will use that method only when there won't be any CVE affected nvrs. So that new ET api call will not give us any binary rpms. And we will just rebuild all builds that are attached to that advisory. So maybe we should leave that method as it is? because it's somehow other use-case then we are working on.

Reply:
You need to keep `get_builds` for odcs `prepare_yum_repo` and `rebuild_images_on_parent_image_build.py` because they're expecting build nvr instead of binary nvr.

So you need another api to get all binary rpm nvrs from advisory, and for the non-RHSA case, you also need binary rpm 
nvrs instead of build/srpm nvrs, because we'll use these nvrs later to query/filter images in lightblue for both the RHSA and non-RHSA cases.

Reply:
So I've changed that part. You can check it now. And btw I left some comments in the code with some stuff that is not pretty clear for me, so could you comment on them too? Thx)